inference-engine/tests/functional/plugin/shared/src/single_layer_tests/log_softmax.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/log_softmax.hpp"
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+
+#include "ie_core.hpp"
+
+#include "ngraph/op/log_softmax.hpp"
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+
+namespace LayerTestsDefinitions {
+
+std::string LogSoftmaxLayerTest::getTestCaseName(testing::TestParamInfo<logSoftmaxLayerTestParams> obj) {
+    InferenceEngine::Precision netPrecision;
+    InferenceEngine::Precision inPrc, outPrc;
+    InferenceEngine::Layout inLayout, outLayout;
+    InferenceEngine::SizeVector inputShape;
+    int64_t axis;
+    std::string targetDevice;
+    std::map<std::string, std::string> config;
+    std::tie(netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, axis, targetDevice, config) = obj.param;
+
+    std::ostringstream result;
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "inPRC=" << inPrc.name() << "_";
+    result << "outPRC=" << outPrc.name() << "_";
+    result << "inL=" << inLayout << "_";
+    result << "outL=" << outLayout << "_";
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "axis=" << axis << "_";
+    result << "trgDev=" << targetDevice;
+
+    return result.str();
+}
+
+void LogSoftmaxLayerTest::SetUp() {
+    InferenceEngine::SizeVector inputShape;
```

Comment:
I think it's better not to do it. Because Softmax axis must be positive integer value, although LogSoftmax can be negative by according to the [spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/LogSoftmax_5.md).
So _softMaxLayerTestParams_ may looks like it:
```
    using logSoftmaxLayerTestParams = std::tuple<
        InferenceEngine::Precision,         // netPrecision
        InferenceEngine::Precision,         // Input precision
        InferenceEngine::Precision,         // Output precision
        InferenceEngine::Layout,            // Input layout
        InferenceEngine::Layout,            // Output layout
        InferenceEngine::SizeVector,        // inputShape
        int64_t,                            // axis
        std::string,                        // targetDevice
        std::map<std::string, std::string>  // config
>;
```
In my opinion, using _int64_t_ is not good idea for SoftMax.
Moreover, code duplication will still be (but in one file) - we will have to write different tests params for this layers because different axises range of values.
So should me unit these tests?